src/typing/typingSelectors.js

```diff
@@ -25,6 +25,6 @@ export const getCurrentTypingUsers = createSelector(
       return undefined;
     }
 
-    return currentTyping.map(userId => getUserById(users, userId));
+    return currentTyping.userIds.map(userId => getUserById(users, userId));
```

Comment:
Hmmm, is this a bug fix?

Reply:
Now we have `userIds` array and `time` at which event have arrived in `currentTyping`.